core/src/main/java/io/aiven/klaw/service/ExportImportDataService.java

```diff
@@ -0,0 +1,157 @@
+  public KwAdminConfig getAdminConfig(HandleDbRequests handleDbRequests) {
+    log.info(
+        "Selecting Kw Admin Config (tenants, clusters, environments, roles, permissions, teams, users, properties) --- STARTED");
+    List<UserInfo> userList = getUpdatedUserList(handleDbRequests.selectAllUsersAllTenants());
+
+    KwAdminConfig kwMetadata =
+        KwAdminConfig.builder()
+            .tenants(handleDbRequests.getTenants())
+            .clusters(handleDbRequests.getClusters())
+            .environments(handleDbRequests.selectEnvs())
+            .rolesPermissions(handleDbRequests.getRolesPermissions())
+            .teams(handleDbRequests.selectTeams())
+            .users(userList)
+            .properties(handleDbRequests.selectKwProperties())
+            .productDetails(
+                handleDbRequests.selectProductDetails("Klaw").orElse(new ProductDetails()))
+            .build();
+    log.info("Selecting Kw Admin Config --- ENDED");
+    return kwMetadata;
+  }
+
+  private List<UserInfo> getUpdatedUserList(List<UserInfo> userList) {
+    log.info("Users password exported as : {}", PWD_ALL_USERS);
+    BasicTextEncryptor textEncryptor = new BasicTextEncryptor();
+    textEncryptor.setPasswordCharArray(encryptorSecretKey.toCharArray());
+    userList.forEach(user -> user.setPwd(textEncryptor.encrypt(PWD_ALL_USERS)));
+
+    return userList;
+  }
+
+  // Get core configuration of topics, acls, schemas, connectors
+  public KwData getKwData(HandleDbRequests handleDbRequests) {
+    log.info("Selecting Kw Data (topics, acls, schemas, connectors) --- STARTED");
+    KwData kwMetadata =
+        KwData.builder()
+            .topics(handleDbRequests.getAllTopics())
+            .subscriptions(handleDbRequests.getAllSubscriptions())
+            .schemas(handleDbRequests.selectAllSchemas())
+            .kafkaConnectors(handleDbRequests.getAllConnectors())
+            .build();
+    log.info("Selecting Kw Data --- STARTED");
+    return kwMetadata;
+  }
+
+  // Get requests data and activity log
+  public KwRequests getRequestsData(HandleDbRequests handleDbRequests) {
+    log.info(
+        "Selecting Kw Requests Data (topic, subscription, schema and connector requests, activity log)--- STARTED");
+    KwRequests kwMetadata =
+        KwRequests.builder()
+            .topicRequests(handleDbRequests.getAllTopicRequests())
+            .subscriptionRequests(handleDbRequests.getAllAclRequests())
+            .schemaRequests(handleDbRequests.getAllSchemaRequests())
+            .connectorRequests(handleDbRequests.getAllConnectorRequests())
+            .activityLogs(handleDbRequests.getAllActivityLog())
+            .build();
+    log.info("Selecting Kw Requests Data --- ENDED");
+    return kwMetadata;
+  }
+
+  private File getFile(String fileName, String timeStamp) {
+    File file =
+        new File(klawExportFilePath + FILE_PREFIX + "-" + fileName + "-" + timeStamp + FILE_EXT);
+    log.info("File : {}", file);
```

Comment:
Do you need a File.Separator between klawExportFilePath & FilePrefix?
e.g.
target/admin_config-admin-300030303.json

It's actually just the default that needs the / at the end as its been added to the application.properties with the additional '/'

Reply:
Good one. added.